include/networkit/community/ParallelLeiden.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef NETWORKIT_COMMUNITY_PARALLEL_LEIDEN_HPP_
+#define NETWORKIT_COMMUNITY_PARALLEL_LEIDEN_HPP_
+
+#include <atomic>
+#include <condition_variable>
+#include <iomanip>
+#include <iostream>
+#include <mutex>
+#include <omp.h>
+#include <thread>
+#include <networkit/Globals.hpp>
+#include <networkit/auxiliary/Parallel.hpp>
+#include <networkit/auxiliary/Parallelism.hpp>
+#include <networkit/auxiliary/SignalHandling.hpp>
+#include <networkit/auxiliary/Timer.hpp>
+#include <networkit/coarsening/ParallelPartitionCoarsening.hpp>
+#include <networkit/community/CommunityDetectionAlgorithm.hpp>
+#include <networkit/community/Modularity.hpp>
+#include <networkit/community/PLM.hpp>
+#include <networkit/graph/Graph.hpp>
+#include <networkit/structures/Partition.hpp>
+
+using namespace NetworKit;
```

Comment:
`ParallelLeiden` is already defined within the `NetworKit` namespace, this line should not be necessary.